app/styles/components/_panel.scss

```diff
@@ -86,6 +86,19 @@ $panel-content-max-width: 500px;
   background-color: $Theme-color-white;
 }
 
+.Panel--solidLightBlue {
+  background-color: $Theme-color-light-blue;
+}
+
+.Panel--lightBlue {
+  color: $Theme-color-white;
+
+  .Panel-titleImage {
```

Comment:
I don't think these should be nested. Since you are using a modifier, it's not expected to have any changes on other elements.